packages/sdk/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@microsoft/teamsfx",
-  "version": "0.1.2-rc.4",
+  "version": "0.1.3",
```

Comment:
Does SDK have any changes that need to bump version?

Reply:
I don't have the knowledge of this. May have a offline discussion first.